cmd/podman/exists.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"github.com/containers/libpod/libpod/adapter"
```

Comment:
gofmt ok with this here instead of below near line 7?

Reply:
yes but worth tidying up, ty